File: core-java-practice/gcr-codebase/PowerCalculation.java
import java.util.*;

public class PowerCalculation {
    public static void main(String[] args) {
        //read input
        Scanner sc=new Scanner(System.in);

        System.out.println("Enter the base");
        double base=sc.nextDouble(); //take base as input

        System.out.println("Enter the exponent");
        double exponent=sc.nextDouble(); //take exponent as input

        //calculate the power
        double power=Math.pow(base,exponent);

        System.out.println("The solution of power calculation is: " + power);

    }
}

File: core-csharp-practice/gcr-codebase/programming-elements/PowerCalculation.cs
using System;

class PowerCalculation{
    public static void Main(string[] args){

        Console.Write("Enter the base: ");
        double baseVal=double.Parse(Console.ReadLine()); //take base as input

        Console.Write("Enter the exponent: ");
        double exponent=double.Parse(Console.ReadLine()); //take exponent as input

        //power calculation
        double power=Math.Pow(baseVal,exponent);

        //printing the statement
        Console.WriteLine("The calculated power is: "+ power);
    }
}